### PR TITLE
fix(quantic): Updated the way of setting an internal property in the Quantic Refine Toggle

### PR DESCRIPTION
**Pull request by mmitiche**
[SFINT-4539](https://coveord.atlassian.net/browse/SFINT-4539)
### The issue:
The issue here is quite subtle, its goal is to avoid confusion and provide an easy experience for the customers who want to customize The Quantic Refine Toggle component.
This issue was raised because two exposed properties of the QuanticRefneToggle component, the `fullScreen` property  and the `hideSort` property, were used and fed with the same name to internal components of the QuanticRefineToggle in its HTML template. Which was considered as something that can cause some issues for users customizing the component.

### The solution:
To improve this behaviour, the mentioned properties are removed from the HTML template and are instead forced to their correct values in the JS file of the QuanticRefineToggle component.

### CODE REVIEW

**Review by nathanlb — APPROVED** [review]
I don't really understand why this helps but i don't mind doing it this way either.

**Review by erocheleau — COMMENTED** [review]
I would also advocate for proper getter method

**Review by erocheleau — APPROVED** [review]

File: packages/quantic/force-app/main/default/lwc/quanticRefineToggle/quanticRefineToggle.js [hunk 1]
```diff
@@ -128,6 +122,11 @@ export default class QuanticRefineToggle extends LightningElement {
     Object.keys(registeredFacets).forEach((facetId) => {
       this.renderedFacets[facetId] = true;
     });
+
+    const modal = this.getModal();
+    const modalContent = this.getModalContent();
```
[the remark this answers is not in the document]
Comment (mmitiche) [reply packages/quantic/force-app/main/default/lwc/quanticRefineToggle/quanticRefineToggle.js:127]: I'll update that👌

**Comment by mmitiche** [comment]
> I don't really understand why this helps but i don't mind doing it this way either.

Neither do I really x) but I'm trusting our QA on this recommendation.

**Pull request merged**